src/edu/ucsb/nceas/metacat/EventLog.java

```diff
@@ -61,7 +61,7 @@ public class EventLog {
     private static final int USERAGENTLENGTH = 512;
     private EventLogFilter filter = null;
     private boolean enableEventLogIndex = false;
-    private boolean disableEventLog = false;
+    private boolean enableEventLog = true;
```

Comment:
I'm really sorry I didn't catch this last time - could we use the accepted convention for boolean variables and call this `isEventLogEnabled` instead?

("enableEventLog" is a verb phrase - it sounds like you're calling a method :-)